README.md

```diff
@@ -70,3 +70,21 @@ Then view <http://localhost:8080>.
 
 Please follow these naming and spacing conventions when submitting a pull request:
 [React + Redux Conventions](https://unbug.gitbooks.io/react-native-training/content/45_naming_convention.html).
+
+
+## Code style and format
+
+We use a combination of [Prettier](https://prettier.io/docs/en/index.html) and [Eslint](https://eslint.org/docs/user-guide/getting-started). Prettier is an opinionated code formatter but does not care about code-quality rules. It only concerns formatting rules. This is why we use Eslint for code-quality rules but not for formatting rules. Read more about the difference between linters and Prettier here: <https://prettier.io/docs/en/comparison.html>. Not ever do we want formatting rules in the `.eslintrc` configuration file. Not implicitly or explicitly. To make sure of that we have a NPM script called verify-eslint-rules (`npm run verify-eslint-rules`) that gives an error if there are Eslint rules somewhere in our `.eslintrc` configuration that conflicts with Prettiers formatting rules.
+
+A PR will not be merged until it passes all the Eslint rules and is properly formatted with Prettier.
```

Comment:
@Spetastian How about we say, "**Before submitting PRs, please format your code using `npm run lint:fix`**"?